src/components/Comments/Comment.js

```diff
@@ -38,6 +46,7 @@ const Comment = ({
   onLongPress,
   updateVote
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new ref `refSheet` is created using `React.useRef()`. This seems to be used for controlling the visibility of the bottom sheet menu. No issues found here.
```diff
+ const refSheet = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->